mega-service-agentQnA/backend/routes/query.py

```
@@ -0,0 +1,52 @@
+from fastapi import APIRouter, HTTPException, Depends
```

Ruff (0.8.2)

`fastapi.Depends` imported but unused

Remove unused import: `fastapi.Depends`

(F401)